client/src/app/site/polls/components/poll-form/poll-form.component.spec.ts

```diff
@@ -4,7 +4,7 @@ import { E2EImportsModule } from 'e2e-imports.module';
 
 import { PollFormComponent } from './poll-form.component';
 
-describe('PollFormComponent', () => {
+fdescribe('PollFormComponent', () => {
```

Comment:
Is the fdescribe on purpose? Should probably be reverted if not to not hassle woth debugging other tests.

Reply:
no its not :D thanks

Reply:
#5238 is already on top of this PR and has the tests functional again.